ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-undef */
+const { test, expect } = require('@playwright/test');
+const { exec, execSync } = require('child_process');
+const { checkEnv } = require('../helpers/general');
+const {
+  authenticateBoundaryCli,
+  checkBoundaryCli,
+} = require('../helpers/boundary-cli');
+const {
+  createNewOrg,
+  createNewProject,
+  createNewHostCatalog,
+  createNewHostSet,
+  createNewHostInHostSet,
+  createNewTarget,
+  addHostSourceToTarget,
+} = require('../helpers/boundary-ui');
+
+test.use({ storageState: './tests/e2e/artifacts/authenticated-state.json' });
+
+test.beforeAll(async () => {
+  await checkEnv([
+    'E2E_TARGET_IP',
+    'E2E_SSH_USER',
+    'E2E_SSH_KEY_PATH',
+    'E2E_SSH_PORT',
+  ]);
+
+  await checkBoundaryCli();
+});
+
+test('Cancel a session', async ({ page }) => {
```

Comment:
```suggestion
test('Verify session created then cancel the session', async ({ page }) => {
```
Small suggestion. It seems this is really wanting to confirm that a session was created & we could cancel it.